dashboard/test/ui/features/teacher_tools/teacher_dashboard/teacher_homepage_v2.feature

```diff
@@ -0,0 +1,10 @@
+@no_mobile
+Feature: Using the teacher homepage
+  @eyes
+  Scenario: Teacher can view sections on new teacher homepage
+    When I open my eyes to test "teacher homepage"
+    Given I am a teacher
+    Then I use a cookie to mock the DCDO key "teacher-homepage-v2" as "true"
```

Comment:
I think we should create at least one section with student and curriculum assigned so that we are testing the section cards as well. What do you think?